client/src/index.css

```diff
@@ -41,38 +43,37 @@
   text-align: center;
 }
 
-.landing-button {
```

Comment:
making buttons more resuable with just distinctive class names, like boostrap style

ideally, we could style commonly-used elements like buttons by adding class names "button-primary button-large button-hover-primary" etc